src/applications/financial-status-report/pages/expenses/repayments/installmentContracts.js

```diff
@@ -13,16 +16,17 @@ export const uiSchema = {
     </>
   ),
   questions: {
-    hasRepayments: {
-      'ui:title':
-        'Do you make monthly payments on any installments contracts or other debts you make monthly payments on?',
-      'ui:webComponentField': YesNoField,
-      'ui:required': () => true,
-      'ui:errorMessages': {
+    hasRepayments: yesNoUI({
+      title:
+        'Do you make monthly payments on any installment contracts or other debts you make monthly payments on?',
```

Comment:
Change this to say: **Do you make monthly payments on any installment contracts or other debts?**